Cython/Shadow.pyi

```diff
@@ -100,3 +103,19 @@ class typedef(CythonType, Generic[_T]):
 #    def __init__(self) -> None: ...
 
 #def fused_type(*args: Tuple[_T]) -> Type[FusedType[_T]]: ...
+
+class _EmptyDecoratorAndManager(object):
+    def __call__(self, x: _T) -> _T: ...
+
+    def __enter__(self) -> None: ...
+
+    def __exit__(
+        self,
+        exc_type: Optional[Type[BaseException]],
+        exc: Optional[BaseException],
+        tb: Optional[TracebackType]
+    ) -> None: ...
+
+cclass = ccall = cfunc = _EmptyDecoratorAndManager()
+
+def declare(t: Optional[Callable[..., _T]] = None, value: Any = ..., **kwds: Any) -> Optional[_T]: ...
```

Comment:
Maybe spread this between several lines?

Reply:
Yeah wasn't sure, it was just barely on the limit, I'll do that.